decorator.rb

```diff
@@ -0,0 +1,23 @@
+# Base Decorator class
+require_relative 'person'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+end
+
+# CapitalizeDecorator class
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+# TrimmerDecorator class
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0..9]
+  end
+end
```

Comment:
[OPTIONAL] Please create one file called `capitalize_decorator.rb` file and move line no. 12 to 16 in that newly created file :slightly_smiling_face: 

[OPTIONAL] Please create one file called the `trimmer_decorator.rb` file and move line no. 19 to 23 in that newly created file :slightly_smiling_face: